lib/src/base-styles/_nx-colors.scss

```diff
@@ -203,6 +207,8 @@
 
     --nx-color-site-background: var(--nx-swatch-indigo-15);
 
+    --nx-color-interactive-background-hover: var(--nx-swatch-indigo-10);
+    --nx-color-interactive-background-selected: var(--nx-swatch-purple-30);
```

Comment:
This group of properties is already present below, and the hover one is duplicated there

Reply:
> This group of properties is already present below, and the hover one is duplicated there

Good catch: 2e28079a218dc08e6a4518bab2982d42c047d29b